robot/power.py

```diff
@@ -43,6 +56,15 @@ def power_off(self):
         """
         self._send_and_receive({'power': False})
 
+    def set_output(self, output: PowerOutput, value: bool) -> None:
+        """
+        Turn off individual power output.
```

Comment:
This does more than just turning them off -- you can turn them _on_ too.

Also Maybe add "an"?

"Turn on or off an individual power output".